lambda/spec/ssmlUtilSpec.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const ssmlUtil = require('../src/ssmlUtil');
+
+describe('wrapDomain', function() {
+    it('should be wrapped in the domain ssml tag', function() {
+        const speakOutput = "Hello there, I'm a cactus.";
+
+        const actual = ssmlUtil.wrapDomain(speakOutput);
+        const expected = '<amazon:domain name="long-form">Hello there, I\'m a cactus.</amazon:domain>';
```

Comment:
nit, use the speakOutput var with `` syntax

Reply:
fixed